### ISSUE TITLE
Hide redundant information from assistive technologies

### ISSUE THREAD

**Issue by audrasjb** [piwik/piwik#19807]
## Expected Behavior

On several places of the admin, assistive technologies users will get redundant information due to duplicate info between text and title attributes, etc.

## Current Behavior
For example, it's the case on the Visits overview dashboard widget. 
I can open a new PR for each occurrence I find. Hopefully it will help removing those redundant info.
For example, on the Visits Overview widget we currently have:
```
<span class="metricEvolution" title="3 visits in 2022-10-02 compared to 8 visits in 2022-10-01. Evolution: -62,5%">
    <img style="padding-right:4px" src="plugins/MultiSites/images/arrow_down.png">
    <strong class="negative-evolution">-62,5&nbsp;%</strong>
</span>
```

## Possible Solution
`aria-hidden="true"` can be used to hide redundant info from assistive technologies.
That's the case when the most complete information for non sighted users is provided by a title attribute: then we can remove other redundant info.
For example, on the Visits Overview widget, we could have:
```
<span class="metricEvolution" title="3 visits in 2022-10-02 compared to 8 visits in 2022-10-01. Evolution: -62,5%">
    <img style="padding-right:4px" src="plugins/MultiSites/images/arrow_down.png" alt="">
    <strong class="negative-evolution" aria-hidden="true">-62,5&nbsp;%</strong>
</span>
```

## Your Environment

* Matomo Version: last release.
* PHP Version: 7.4
* Server Operating System: Linux
* Additionally installed plugins: None

**Comment by heurteph-ei** [comment]
As #19808 and #19809 are merged, and version 4.12.1 now released, I think this can be closed, now...?

**Comment by sgiehl** [comment]
I'll close this one now. If there are further suggestions on how to improve accessibility, feel free to create a new issue.
